test/api/multiClassMatch.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const hljs = require('../../build');
+hljs.debugMode();
```

Comment:
```suggestion
```

Probably want to remove this?

Reply:
Nope, tests should be running in debug mode to fully raise errors.